<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartWifiWidget</class>
 <widget class="QWidget" name="SmartWifiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartWifi</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>SmartWifi</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>601</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>套餐信息</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QStackedWidget" name="serviceInfoStack">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>561</width>
        <height>131</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>套餐类型：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>剩余流量：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>目前使用：</string>
        </property>
       </widget>
       <widget class="QLabel" name="servieceFlowLabel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>流量计费</string>
        </property>
       </widget>
       <widget class="QLabel" name="restFlowLabel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>50</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>300 MB</string>
        </property>
       </widget>
       <widget class="QLabel" name="usedFlowLabel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>90</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>100 MB</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="flowPBar">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>50</y>
          <width>151</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="servieceMonthLabel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>按月计费</string>
        </property>
       </widget>
       <widget class="QLabel" name="overdueLabel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>60</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>2013年13月13日</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>终止时间：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>套餐类型：</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>没有套餐信息，您需要重新订阅。</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="servieceHourLabel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>小时计费</string>
        </property>
       </widget>
       <widget class="QLabel" name="restHourLabel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>50</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>300 Hours</string>
        </property>
       </widget>
       <widget class="QLabel" name="usedHourLabel">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>90</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>100 Hours</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="hourPBar">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>50</y>
          <width>151</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>剩余时间：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>套餐类型：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>目前使用：</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>157</y>
       <width>601</width>
       <height>45</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>路由器状态：</string>
      </property>
     </widget>
     <widget class="QLabel" name="routerStatusLabel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>SmartWifi</string>
     </property>
     <widget class="QPushButton" name="enableBtn">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>启用</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>如果你启用SmartWifi，即可体验飞速的翻墙快感，访问全球任何网页</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>601</width>
       <height>63</height>
      </rect>
     </property>
     <property name="title">
      <string>续费</string>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>可以选择不同套餐，以方便你的使用。</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enableBtn_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>续费</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>白名单</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>正在测试连接...</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>321</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>配置</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>路由器IP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>自动断开时间</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
